### PR TITLE
Writer

### PR DESCRIPTION
**Pull request by mhosken**
This adds glyph range support to fea2fea for #772.

I hate git. Sorry this isn't such a clean PR :(

### CODE REVIEW

**Comment by anthrotype** [comment]
oh sorry the merge conflict is because we merged to master the whitespace/linting PR #777 

You need to do this from your `writer` branch, provided that `upstream` is the name you gave to the `fonttools/fonttools` remote:

```shell
git fetch upstream
git rebase upstream/master
git push --force
```

**Comment by anthrotype** [comment]
Or I can fix it manually if you want.

**Comment by mhosken** [comment]
Please can you fix it. It's giving me the run around. If I click resolve conflicts in this PR I just get a spinning wheel :(

**Comment by anthrotype** [comment]
sure no problem. Can you please confirm it is the last three commits which you'd like to add to master?

**Comment by mhosken** [comment]
Yes last 3 commits (well commits -3, -2, since -1 should be no different)

**Comment by anthrotype** [comment]
Ok, i'll do that.

File: Lib/fontTools/feaLib/testdata/spec9e.fea [hunk 1]
```diff
@@ -1,4 +1,6 @@
 table name {
-  nameid 9 "Joachim M\00fcller-Lanc\00e9";    # Windows (Unicode)
-  nameid 9 1 "Joachim M\9fller-Lanc\8e";      # Macintosh (Mac Roman)
+#test-fea2fea: nameid 9 "Joachim Müller-Lancé";
```
Comment (anthrotype) [inline Lib/fontTools/feaLib/testdata/spec9e.fea]: Shouldn't we escape non ASCII characters in the fea output?

Comment (mhosken) [inline Lib/fontTools/feaLib/testdata/spec9e.fea]: Yes. Feel free to add the processing (I would ignore the Mac encoding stuff, and just go with Unicode). Note that \ needs to be encoded as \005C. If you want me to do this, I'm happy to do this early next year. If so, raise a bug.

**Comment by mhosken** [comment]
sorry about that. That'll teach me to tidy up!

**Comment by anthrotype** [comment]
merged manually #779

**Pull request closed without merging**